src/algebra/order/ring/defs.lean

```diff
@@ -327,6 +327,22 @@ lemma mul_le_mul_of_nonpos_of_nonpos' (hca : c ≤ a) (hdb : d ≤ b) (ha : a 
   a * b ≤ c * d :=
 (mul_le_mul_of_nonpos_left hdb ha).trans $ mul_le_mul_of_nonpos_right hca hd
 
+/-- Variant of `mul_le_of_le_one_left` -/
+lemma le_mul_of_le_one_left (hb : b ≤ 0) (h : a ≤ 1) : b ≤ a * b :=
+by simpa only [one_mul] using mul_le_mul_of_nonpos_right h hb
+
+/-- Variant of `le_mul_of_one_le_left` -/
```

Comment:
```suggestion
/-- Variant of `le_mul_of_one_le_left` for nonpositive something -/
```
to avoid people having to read the entire type to distinguish from `mul_le_of_le_one_left`.